### DISCUSSION TITLE
Is Push Protection GA?

### DISCUSSION THREAD

**Discussion by AnthonyHerman** [community/events#36] [Security]
I've been trying to figure this out for awhile. Is push protection GA or not? It seems like it is but the docs still show this banner:
![image](https://user-images.githubusercontent.com/46350435/200918393-6d20500c-45ce-4fb8-a426-a6ebabdcf839.png)
(https://docs.github.com/en/enterprise-cloud@latest/code-security/secret-scanning/protecting-pushes-with-secret-scanning)

**Comment by chrisreddington** [comment]
Hey there @AnthonyHerman, thanks for the question. I've checked in with the team, and can confirm that the beta label in the docs is still accurate. Hope it helps!
